shared/test/dataDriver_test.js

```diff
@@ -21,6 +21,10 @@ describe("loop.DataDriver", () => {
     driver = new loop.DataDriver({
       dispatcher
     });
+    let originalConnect = driver._connectToRoom;
+    sandbox.stub(driver, "_connectToRoom", token => {
+      originalConnect.call(driver, token);
+    });
```

Comment:
Just because I was looking at how the other code does it. I think it is a great idea to only stub it for one test.